src/section.js

```diff
@@ -81,23 +81,14 @@ class Section {
 	 * @param  {method} _request a request method to use for loading
 	 * @return {string} output a serialized XML Document
 	 */
-	render(_request){
+	render(){
 		var rendering = new defer();
 		var rendered = rendering.promise;
 		this.output; // TODO: better way to return this from hooks?
 
-		this.load(_request).
+		this.load().
 			then(function(contents){
-				var serializer;
```

Comment:
@fchasen Hi, reason why we have removed the serializer is because if we are trying to serialize the xml, we have issues like self closing script tags(which causes issues in IE as it does not treat them well and xhtml is not rendered at all).
To adress this we are directly using **response.responseText**. Problem using **response.responseXML** is that it self closes script tags becasue of the xml structure.
Please suggest if you have any other approach or thoughts on this.